src/components/common/YourCompany/ChangeBusinessType.vue

```diff
@@ -286,10 +286,12 @@ export default class ChangeBusinessType extends Mixins(CommonMixin) {
   confirmArticles = false
   isEditingType = false
   dropdown: boolean = null
+  supportedEntityTypes = null
```

Comment:
If you initialize this as `[] as Array<string>` then you don't need conditional chaining on line 324 and you'll have typing :)